README.rst

```diff
@@ -90,7 +89,6 @@ They are kept in separate repos and updated manually by coping:
 * removenbsp
 * `structuredheadings <https://github.com/PolicyStat/ckeditor-plugin-structured-headings>`_
 
-These plugins packed together with CKEditor by symlinking during  update_ckeditor.py run.
```

Comment:
does `update_ckeditor.py` need to be updated to not symlink them now?

Reply:
Yes. It's removed https://github.com/PolicyStat/PolicyStat/pull/4601/files#diff-6e6b478860ccd1b74329c46ffc3739a2L37